gno.land/pkg/gnoclient/client_queries.go

```diff
@@ -177,3 +182,52 @@ func (c *Client) LatestBlockHeight() (int64, error) {
 
 	return status.SyncInfo.LatestBlockHeight, nil
 }
+
+// GetTransaction retrieves the transaction details for a given transaction hash
+// The provided hash must be a valid base64 encoded string
+func (c *Client) Transaction(hash string) (*ctypes.ResultTx, error) {
```

Comment:
The comment says "GetTransaction". Should the function name be `GetTransaction`?